Symfony/CS/ConfigInterface.php

```diff
@@ -59,6 +59,8 @@ public function getFixers();
      * Sets the root directory of the project.
      *
      * @param string $dir The project root directory
+     *
+     * @return Config The same instance
```

Comment:
@keradus `$this` in return type annotation is **not** a variable. It means _same instance_, and most of good IDEs (i.e. PhpStorm) recognize it very well. For worst IDEs you can add a chain `ConfigInterface|$this`